test/AffInterface.purs

```diff
@@ -53,6 +41,8 @@ addr1 :: OgmiosAddress
 addr1 =
   "addr1qyc0kwu98x23ufhsxjgs5k3h7gktn8v5682qna5amwh2juguztcrc8hjay66es67ctn0jmr9plfmlw37je2s2px4xdssgvxerq"
 
+type TestPlanM a = MoteT (Const Void) (QueryM Unit) Aff a
```

Comment:
Also I think it's a bit confusing to have two `TestPlanM`s. Instead of having two (or more), why not parameterize `TestM.TestM` by some `m :: Type -> Type`?